submissions/IhorOzerov/html-css-popup/index.html

```diff
@@ -0,0 +1,157 @@
+                            <img class="visible-img" src="icons/contacts.png"alt=""/>
+                            <span>Contacts</span>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="menu-a" href="#"> 
+                            <img class="visible-img" src="icons/documents.png" alt=""/>
+                            <span>Documents</span>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="menu-a" href="#">
+                            <img class="visible-img" src="icons/files.png"alt=""/>
+                            <span>Files</span>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="menu-a" href="#"> 
+                            <img class="visible-img" src="icons/mail.png"alt=""/>
+                            <span>Mail</span>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="menu-a" href="#"> 
+                            <img class="visible-img" src="icons/maps.png"alt=""/>
+                            <span>Maps</span>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="menu-a" href="#"> 
+                            <img class="visible-img" src="icons/mars.png"alt=""/>
+                            <span>Mars</span>
+                        </a>
+                    </li>
+                    <li class="more-btn greatest">
+                        <input id="more" type="checkbox" class="visually-hidden">
+                        <label class="more-word" for="more">More</label>
+                        <ul class="opened">
+                            <li>
+                                <a class="menu-a" href="#"> 
+                                    <img class="visible-img" src="icons/news.png"alt=""/>
+                                    <span>News</span>
+                                </a>
+                            </li>
+                            <li>
+                                <a class="menu-a" href="#">
+                                    <img class="visible-img" src="icons/photos.png"alt=""/>
+                                    <span>Photos</span>
+                                </a>
+                            </li>
+                            <li>
+                                <a class="menu-a" href="#"> 
+                                    <img style="height: 60px; width:60px" class="visible-img" src="icons/promo.png"alt=""/>
+                                    <span>Promo</span>
+                                </a>
+                            </li>
+                            <li>
+                                <a class="menu-a" href="#"> 
+                                    <img style="height: 50px;" class="visible-img" src="icons/search.png" alt=""/>
```

Comment:
Avoid using inline styling.
Separate responsibilities: HTML for markup, CSS for styles, and JS for logic. [Read this article](https://jakearchibald.com/2022/img-aspect-ratio/) and find out how to save image aspect-ratio